airbyte-ci/connectors/auto_merge/src/auto_merge/pr_validators.py

```diff
@@ -48,12 +48,13 @@ def head_commit_passes_all_required_checks(
     return True, None
 
 
-# A PR is considered auto-mergeable if:
+# A standard PR is considered auto-mergeable if:
```

Comment:
nit: since we don't define what "standard" means here, I'd prefer to leave this comment as-is.

Reply:
Reverted.